src/components/SEP/Proposals/SEPProposalsAndAssignmentsTable.tsx

```diff
@@ -44,6 +44,64 @@ type SEPProposalsAndAssignmentsTableProps = {
   selectedCallId: number;
 };
 
+const getGradesFromAssignments = (assignments: SEPProposalAssignmentType[]) =>
+  assignments?.map((assignment) => assignment.review?.grade) ?? [];
+
+const SEPProposalColumns = [
+  { title: 'ID', field: 'proposal.proposalId' },
+  {
+    title: 'Title',
+    field: 'proposal.title',
+  },
+  {
+    title: 'Status',
+    field: 'proposal.status.name',
+  },
+  {
+    title: 'Date assigned',
+    field: 'dateAssigned',
+    render: (rowData: SEPProposalType): string =>
+      dateformat(new Date(rowData.dateAssigned), 'dd-mmm-yyyy HH:MM:ss'),
```

Comment:
We should probably add this task https://jira.esss.lu.se/browse/SWAP-2345 to the next sprint